### PR TITLE
Huly v295 update

### PR DESCRIPTION
**Pull request by aonnikov**

### CODE REVIEW

File: template.compose.yaml [hunk 1]
```diff
@@ -57,11 +57,25 @@ services:
...
       - MODEL_ENABLED=*
       - ACCOUNTS_URL=http://${SERVER_ADDRESS}:3000
       - ACCOUNT_PORT=3000
     restart: unless-stopped
+  workspace:
+    image: hardcoreeng/workspace:${HULY_VERSION}
+    links:
+      - mongodb
+      - minio
+    environment:
```
Comment (lexiv0re) [inline template.compose.yaml:69]: Please also add NOTIFY_INBOX_ONLY=true

File: kube/workspace/workspace-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+              valueFrom:
+                configMapKeyRef:
+                  name: huly-config
+                  key: MONGO_URL
+            - name: SERVER_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: huly-secret
+                  key: SERVER_SECRET
+            - name: TRANSACTOR_URL
```
Comment (lexiv0re) [inline kube/workspace/workspace-deployment.yaml:48]: Please also add NOTIFY_INBOX_ONLY=true

**Review by lexiv0re — APPROVED** [review]

File: template.compose.yaml (hunk 1 above)
Comment (aonnikov) [reply template.compose.yaml:67]: done

File: kube/workspace/workspace-deployment.yaml (hunk 2 above)
Comment (aonnikov) [reply kube/workspace/workspace-deployment.yaml:48]: done

File: template.compose.yaml (hunk 1 above)
Comment (lexiv0re) [reply template.compose.yaml:67]: Cannot see the change here. Did you forget to commit/push? :)

File: kube/workspace/workspace-deployment.yaml (hunk 2 above)
Comment (lexiv0re) [reply kube/workspace/workspace-deployment.yaml:48]: Cannot see the change there. Did you forget to commit/push?

Comment (aonnikov) [reply kube/workspace/workspace-deployment.yaml:48]: added now

**Pull request merged**